main.py

```diff
@@ -1,7 +1,45 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
+
+genres = ["Western", "Action", "Dramma"]
+actors = [("George", "Klooney"),
```

Comment:
There is a typo in the actor's name 'Klooney'. It should be 'Clooney' to match the corrected name later in the code.